test/functional/JIT_Test/testng.xml

```diff
@@ -548,5 +548,13 @@
       <class name="jit.test.vich.ReadTest" />
     </classes>
   </test>
+
+  <!-- jit.test.recognizedMethod tests start here -->
+  <test name="RecognizedSqrtTest">
```

Comment:
We can generalize this name to something like "RecognizedMethodTests", since this group will have other recognized methods added to it moving forward. Do you agree that we should generalize the name?

Reply:
Agree. I am going to change it.